app/Controllers/extensionController.php

```diff
@@ -40,7 +42,7 @@ public function indexAction(): void {
 
 	/**
 	 * Fetch extension list from GitHub
-	 * @return list<array{name:string,author:string,description:string,version:string,entrypoint:string,type:'system'|'user',url:string,method:string,directory:string}>
+	 * @return list<ExtensionFullMetadata>
```

Comment:
```suggestion
	 * @phpstan-return list<ExtensionFullMetadata>
```

When it is something only supported by PHPStan